forte/data/extractor/base_extractor.py

```diff
@@ -257,16 +257,25 @@ def extract(self, pack: DataPack,
         """
         pass
 
-    def remove_from_pack(self, pack: DataPack, instance: Annotation):
+    def pre_evaluation_action(self, pack: DataPack,
+                instance: Annotation):
         r"""Functionality: Remove the existing feature of the instance
         in a pack. We might remove an attribute under an entry or remove
         the entry itself directly, which will depend on the different
```

Comment:
Removing attribute is just one of the possible actions, you can say when this function will be called, and then say what are the possible actions that could happen here.

Reply:
Got it.